surfactant/cmd/plugin.py

```diff
@@ -14,3 +15,74 @@ def plugin_list_cmd():
         print(f"canonical name: {pm.get_canonical_name(p)}")
         plugin_name = pm.get_name(p) if pm.get_name(p) else ""
         print(f"name: {plugin_name}")
```

Comment:
```suggestion
        print(f"\t> canonical name: {pm.get_canonical_name(p)}")
        plugin_name = pm.get_name(p) if pm.get_name(p) else ""
        print(f"\t  name: {plugin_name}")
```
I'm wondering if maybe indenting the list of plugins and removing the `print("-------")` separators will make the output more legible.